  pch.cpp
  MyRestrauntDiary.cpp
  MyRestrauntDiaryDlg.cpp
  코드를 생성하고 있습니다...
MyRestrauntDiaryDlg.obj : error LNK2019: _mysql_error@4"public: int __thiscall CMyRestrauntDiaryDlg::setDataToList(void)" (?setDataToList@CMyRestrauntDiaryDlg@@QAEHXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyRestrauntDiaryDlg.obj : error LNK2019: _mysql_init@4"public: int __thiscall CMyRestrauntDiaryDlg::setDataToList(void)" (?setDataToList@CMyRestrauntDiaryDlg@@QAEHXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyRestrauntDiaryDlg.obj : error LNK2019: _mysql_real_connect@32"public: int __thiscall CMyRestrauntDiaryDlg::setDataToList(void)" (?setDataToList@CMyRestrauntDiaryDlg@@QAEHXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyRestrauntDiaryDlg.obj : error LNK2019: _mysql_query@8"public: int __thiscall CMyRestrauntDiaryDlg::setDataToList(void)" (?setDataToList@CMyRestrauntDiaryDlg@@QAEHXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyRestrauntDiaryDlg.obj : error LNK2019: _mysql_store_result@4"public: int __thiscall CMyRestrauntDiaryDlg::setDataToList(void)" (?setDataToList@CMyRestrauntDiaryDlg@@QAEHXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyRestrauntDiaryDlg.obj : error LNK2019: _mysql_free_result@4"public: int __thiscall CMyRestrauntDiaryDlg::setDataToList(void)" (?setDataToList@CMyRestrauntDiaryDlg@@QAEHXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyRestrauntDiaryDlg.obj : error LNK2019: _mysql_fetch_row@4"public: int __thiscall CMyRestrauntDiaryDlg::setDataToList(void)" (?setDataToList@CMyRestrauntDiaryDlg@@QAEHXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
MyRestrauntDiaryDlg.obj : error LNK2019: _mysql_close@4"public: int __thiscall CMyRestrauntDiaryDlg::setDataToList(void)" (?setDataToList@CMyRestrauntDiaryDlg@@QAEHXZ) 함수에서 참조되는 확인할 수 없는 외부 기호
C:\Program Files\MySQL\MySQL Server 8.0\lib\libmysql.lib : warning LNK4272: 'x64' 라이브러리 컴퓨터 종류가 'x86' 대상 컴퓨터 종류와 충돌합니다.
C:\Users\직업능력개발훈련 401호_24번\source\repos\MyRestrauntDiary\Debug\MyRestrauntDiary.exe : fatal error LNK1120: 8개의 확인할 수 없는 외부 참조입니다.
